scripts/pin_the_slow_way.py

```diff
@@ -25,38 +25,90 @@
 import conda_smithy.feedstocks as feedstocks
 
 
-pinned = {
+# Some links that point to backward compatibility
+# https://abi-laboratory.pro/tracker/timeline/ffmpeg/
+# https://abi-laboratory.pro/tracker/timeline/libtiff/
+# https://abi-laboratory.pro/tracker/timeline/libzip/
+# https://abi-laboratory.pro/tracker/timeline/fontconfig/
+# https://abi-laboratory.pro/tracker/timeline/freetype/
+# https://abi-laboratory.pro/tracker/timeline/openssl/
+# https://abi-laboratory.pro/tracker/timeline/sqlite/
+# https://abi-laboratory.pro/tracker/timeline/qt/
+# https://abi-laboratory.pro/tracker/timeline/giflib/
+# http://upstream.rosalinux.ru/versions/libgeos_c.html
+# http://upstream.rosalinux.ru/versions/libgeos.html
+# http://upstream.rosalinux.ru/versions/netcdf.html
+# http://upstream.rosalinux.ru/versions/readline.html
+# http://upstream.rosalinux.ru/versions/tk.html
+# http://upstream.rosalinux.ru/versions/libsox.html
+# http://upstream.rosalinux.ru/versions/zlib.html
+# http://upstream.rosalinux.ru/versions/ncurses.html
+# http://upstream.rosalinux.ru/versions/blitz.html
+
+build_pin = {
           'boost': 'boost 1.61.*',
           'bzip2': 'bzip2 1.0.*',
           'ffmpeg': 'ffmpeg 2.8.*',
           'fontconfig': 'fontconfig 2.11.*',
           'freetype': 'freetype 2.6.*',
           'geos': 'geos 3.4.*',
-          'giflib': 'giflib 5.1.*',
+          'giflib': 'giflib 5.1.2',
           'hdf5': 'hdf5 1.8.17|1.8.17.*',
           'icu': 'icu 56.*',
           'jpeg': 'jpeg 9*',
           'libblitz': 'libblitz 0.10|0.10.*',
           'libmatio': 'libmatio 1.5.*',
           'libnetcdf': 'libnetcdf 4.4.*',
-          'libpng': 'libpng >=1.6.23,<1.7',
+          'libpng': 'libpng 1.6.21',
           'libsvm': 'libsvm 3.21|3.21.*',
           'libtiff': 'libtiff 4.0.*',
           'ncurses': 'ncurses 5.9*',
           'netcdf-cxx4': 'netcdf-cxx4 4.3.*',
           'netcdf-fortran': 'netcdf-fortran 4.4.*',
           'openblas': 'openblas 0.2.18|0.2.18.*',
-          'openssl': 'openssl 1.0.*',
+          'openssl': 'openssl 1.0.2h',
```

Comment:
Yeah, when I made that comment it was in the run section. GitHub has been doing some funny things with comments when diffs change.